test/box-luatest/gh-6786_func_index_iterator_stable_test.lua

```diff
@@ -57,3 +57,54 @@ g.test_func_index_iterator_stable = function()
         t.assert_equals(s.index.f:select(), {})
     end)
 end
+
+g.before_test('test_func_index_allocator', function()
```

Comment:
I think we should commit the test to the master and backport it to 2.8 and 2.10.

Reply:
done.